configure.ac

```diff
@@ -1611,7 +1611,7 @@ AS_CASE([$host],
 ## shared library support
 AS_IF([$shared_libraries_supported],
   [AS_CASE([$host],
-    [*-*-mingw32|*-pc-windows],
+    [*-*-mingw32|*-pc-windows|*-*-cygwin*],
```

Comment:
This skips the test for `dlopen` on Cygwin and treats it the same as mingw and msvc